### PR TITLE
[HEALTHVIC-1186] - added footer main menu

### PR DESCRIPTION
**Pull request by awset**
## Motivation and Context

**JIRA issue:** https://salsadigital.atlassian.net/browse/HEALTHVIC-1192

## Changed

1. new field to configure footer menu is available in tide
2. FE will use this field, and fallback to main menu as per current implementation to support legacy sites.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [ ] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

File: packages/ripple-nuxt-tide/lib/core/tide.js [hunk 1]
```diff
@@ -98,7 +98,8 @@ export const tide = (axios, site, config) => ({
   getMenuFields: function () {
     return {
       menuMain: 'field_site_main_menu',
-      menuFooter: 'field_site_footer_menu'
+      menuFooter: 'field_site_footer_menu',
+      menuMainFooter: 'field_site_footer_main_menu'
```
Comment (gargsuchi) [inline packages/ripple-nuxt-tide/lib/core/tide.js:102]: Hi @awset
Can we put in a condition here?
If field_site_footer_main_menu is not available, we use field_site_footer_menu as default?

Comment (awset) [reply packages/ripple-nuxt-tide/lib/core/tide.js:102]: no we can't, the condition is available here, https://github.com/dpc-sdp/ripple/pull/1086/files#diff-ee87bd1376d57e8223e58511cb4dfa714906292d41a23013d1e714334c6e36cfR71

Comment (gargsuchi) [reply packages/ripple-nuxt-tide/lib/core/tide.js:102]: Ah - thank you!

**Review by gargsuchi — APPROVED** [review]

**Comment by MdNadimHossain** [comment]
This is currently in the backlog. So closing this PR until the story comes into the sprint. 
### The ticket that asked to close this 
- https://digital-engagement.atlassian.net/browse/SDPA-6121
### Some UX research will be undertake as part of ticket and is currently in the backlog - 
https://digital-engagement.atlassian.net/browse/SDPA-6119
https://digital-engagement.atlassian.net/browse/SDPA-6109

**Comment by kepol** [comment]
@MdNadimHossain or @dylankelly Can we get an update on this one? Thanks.

**Pull request closed without merging**